lib/syndicates.js

```diff
@@ -0,0 +1,367 @@
+'use strict';
+
+const fetch = require('node-fetch');
+const cheerio = require('cheerio');
+const path = require('path');
+const { hash } = require('./utils');
+const readJson = require('./readJson');
+
+const rankRegex = /Rank \d[:]? /i;
+function parseRank(rank) {
+  return rank.replace(rankRegex, '');
+}
+
+function parsePlace(syndicate, section, rank) {
+  let syndicateWithSection = syndicate;
+  if (section) {
+    syndicateWithSection += ` (${section})`;
+  }
+  return [syndicateWithSection, rank].filter(Boolean).join(', ');
+}
+
+function parseStanding(standing) {
+  return Number(standing.replace(' ', '').replace(',', ''));
+}
+
+/**
+ * @typedef {Object} DefaultSyndicateMapperResult
+ * @property {_id} _id Unique hash of the item
+ * @property {string} item The name of the item
+ * @property {number} chance The dropchance of the item
+ * @property {string} rarity The rarity of the item
+ * @property {string} place The location where the item drops
+ * @property {number} standing The amount of standing this item costs
+ */
+
+/**
+ * @typedef {function} DefaultMapper
+ * @param {module:cheerio/CheerioAPI} $ cheerio interface
+ * @param {string} element Current element
+ * @param {string} syndicate Name of the Syndicate
+ * @param {string} [section] Section title - potentially empty
+ * @returns {DefaultSyndicateMapperResult}
+ */
+// Default mapper, maps the most syndicate ware lists.
```

Comment:
Jsdoc on a function should abutt the function with no other comments in between, and you wouldn't need the @typedef line, just move the description in there

Reply:
Ah thanks! First time using jsdoc, haha!